mautrix/client/api/events.py

```diff
@@ -14,8 +14,15 @@
                            MediaMessageEventContent, PresenceState, EventContent, Membership,
                            ReactionEventContent, RelationType, Obj, Serializable)
 from mautrix.types.event.state import state_event_content_map
+from mautrix.util.opt_prometheus import Counter, Histogram
 from .base import BaseClientAPI
 
+API_CALLS = Counter("bridge_matrix_api_calls",
+                    "The number of Matrix client API calls made", ("method",))
+API_CALLS_FAILED = Counter("bridge_matrix_api_calls_failed",
+                           "The number of Matrix client API calls which failed", ("method",))
+API_CALLS_TIME = Histogram("bridge_matrix_api_calls_update",
```

Comment:
Couldn't find a Histogram value like this on the NodeJS bridge repo, so I came up with `bridge_matrix_api_calls_update`.

`_update` is probably not a good name.

Reply:
Renamed in https://github.com/vector-im/mautrix-python/pull/2/commits/6ab8cd5ad9556cac7897aefd008a0f0d745f7502